### PR TITLE
Feat/import export application

### PR DESCRIPTION
**Pull request by PiusKariuki**

### CODE REVIEW

**Comment by PiusKariuki** [comment]
@Michaelndula  Please review this PR

**Review by Michaelndula — COMMENTED** [review]
rename this to our scope @sjthc

File: packages/esm-admin-nav-app/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { defineConfigSchema, getSyncLifecycle } from '@openmrs/esm-framework';
+import { configSchema } from './config-schema';
+import NavbarActionButton from './navbar/navbar-action-button.component';
+
+const moduleName = '@kenyaemr/esm-admin-nav-app';
```
Comment (Michaelndula) [inline packages/esm-admin-nav-app/src/index.ts:5]: rename this our scope @sjthc

File: packages/esm-admin-nav-app/src/navbar/navbar-action-button.test.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    expect(globalActionButton).toHaveClass(
+      'cds--btn--icon-only active cds--header__action cds--btn cds--btn--primary cds--btn--icon-only cds--btn cds--btn--primary',
+    );
+
+    // should render the overlay
+    const searchInput = screen.getByRole('searchbox', { name: 'Search for a module' });
+    expect(searchInput).toBeInTheDocument();
+    const kenyaemrLinks = [
+      'System Info',
+      'KenyaEMR Home',
```
Comment (Michaelndula) [inline packages/esm-admin-nav-app/src/navbar/navbar-action-button.test.tsx:34]: Remove this

File: packages/esm-admin-nav-app/src/navbar/nav-utils.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    requiresAdmin: true,
+  },
+  {
+    label: 'Billable Services',
+    url: `${openmrsSpaBase}billable-services`,
+    icon: <InventoryManagement size={24} />,
+    requiresAdmin: true,
+  },
+  {
+    label: 'Import Export',
```
Comment (Michaelndula) [inline packages/esm-admin-nav-app/src/navbar/nav-utils.tsx:73]: Let's use the module name as Backup

**Comment by PiusKariuki** [comment]
I have resolved this  @Michaelndula

**Pull request merged**